osu.Game.Tests/Visual/Gameplay/TestSceneGameplaySamplePlayback.cs

```diff
@@ -1,8 +1,6 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-#nullable disable
```

Comment:
Enabling NRT in this file seems a bit drive-by (there seem to be no actual changes in this file related to `FrameStableClock`) but ok.

Reply:
I think I originally had changed in this file...